packages/@ourworldindata/grapher/src/mapCharts/GlobeController.ts

```diff
@@ -135,22 +134,43 @@ export class GlobeController {
                 ? interpolateNumber(currentZoom, targetZoom)
                 : undefined
 
-        for (let t = this.tickStep; t <= 1; t += this.tickStep) {
-            // Check if the animation was canceled
-            if (signal.aborted) return
-
-            // animate globe rotation
-            this.globeConfig.rotation = animatedCoords(easeCubicOut(t))
-
-            // animate zoom
-            if (animatedZoom)
-                this.globeConfig.zoom = animatedZoom(easeCubicOut(t))
-
-            await delay(this.msPerTick)
-        }
-
-        // ensure we end exactly at the target values
-        this.globeConfig.rotation = targetCoords
-        if (targetZoom !== undefined) this.globeConfig.zoom = targetZoom
+        const animPromise = new Promise<void>((resolve, reject) => {
+            const now = Date.now()
+            const duration = this.msPerTick * (1 / this.tickStep)
```

Comment:
Okay, arguably this can now be changed to just one `this.animDuration` property, don't need to split it up any more :)